boot-qemu.sh

```diff
@@ -126,19 +126,23 @@ function get_full_kernel_path() {
     [[ -f ${KERNEL} ]] || die "${KERNEL} does not exist!"
 }
 
-# Print QEMU version as a five or six digit number
+# Takes a version (x.y.z) and prints a six or seven digit number
+# For example, QEMU 6.2.50 would become 602050 and Linux 5.10.100
+# would become 510100
+function print_version_code() {
```

Comment:
let's make the function suffix follow the others; `print_ver_code`

Reply:
Sure thing, done!